apps/www/app/docs/sidebar.tsx

```diff
@@ -105,6 +108,35 @@ const MobileMenuButton = chakra("button", {
   },
 })
 
+export const MobileMenuBreadcrumbs = () => {
```

Comment:
Can we use the breadcrumb component for this?
https://github.com/chakra-ui/chakra-ui/blob/743421bba95ab7ff2dd7c751976a170383c7360a/apps/compositions/src/examples/breadcrumb-basic.tsx

Reply:
Done!